docs/wiki/Wallet.md

```diff
@@ -0,0 +1,499 @@
+* **Manage Peers:** Allows adding/removing of network peers.
+* **Wallet Logs:** Opens the log file for the wallet.
+* **Rescan:** Begins a rescan for all relevant transactions involving all active addresses and watched outpoints.
+Rescans can be time-consuming depending on the amount of data that must be checked, and the size of the blockchain.
+* **Disable Wallet:** This function will disable the wallet, it will not be connected to when you start Bison Wallet
+and cannot be used until it is re-enabled. This will also disable all token wallets that depend on this wallet.
+
+# Fee Status
+
+The fee status section contains summary information around the fees associated with the selected asset.
+
+* **Network Fee Rate:** fee rate is the number of units per byte that a user pays to add their transaction 
+to the specified blockchain.
+* **Exchange Rate:** USD exchange rate for the selected asset.
+* **Send Fees:** Minimum fee for transferring funds on the blockchain.
+* **Sell Fees:** Minimum fee for performing a sell trade on the blockchain.
+* **Buy Fees:** Minimum fee for performing a buy trade on the blockchain.
+
+<img src="./images/using-bison-wallet/wallets-feeStatus.png" width="1200" alt="">
+
+# Transaction History
+
+This section shows a list of all transactions that have occurred involving this wallet, including send, receive
+bonds, splits and mixes.
+
+<img src="./images/using-bison-wallet/wallets-txHistory.png" width="1200" alt="">
+
+Clicking on a transaction will open the Transaction Details dialog, which displays
+summary information about the transaction along with a link to a block explorer.
+
+<img src="./images/using-bison-wallet/wallets-txDetail.png" width="1200" alt="">
+
+# Asset Markets
+
+The asset markets section displays the available markets for the selected assets within the connected DEX servers.
+
+<img src="./images/using-bison-wallet/wallets-markets.png" width="1200" alt="">
+
+# Recent Activity
+
+The recent activity section lists the recent order history for the specified asset. 
+
+<img src="./images/using-bison-wallet/wallets-activity.png" width="1200" alt="">
+
+Summary information for each order is displayed in the recent activity section. 
+The [Order Details](Order-Management#order-details) view can be opened by clicking
+the <img src="./images/using-bison-wallet/button-link.png" width="20" alt=""> button. 
+
+<img src="./images/using-bison-wallet/wallets-activity2.png" width="1200" alt="">
+
+# Asset-Specific Features
+
+This section of the wallet view contains functionality specific to the selected asset.
+
+<img src="./images/using-bison-wallet/wallets-assetSpecific.png" width="1200" alt="">
+
+
+## Decred 
+
+
+## Privacy
+
+When privacy is enabled, all of your funds are sent through Decred's address history obfuscation service, using a 
+protocol called StakeShuffle. StakeShuffle generates outputs that cannot be definitively linked to previous on-chain 
```

Comment:
idk did we coin StakeShuffle? I thought CoinShuffle++

Reply:
see #3042

Reply:
Ohhh, we did coin it. Thanks.